packages/fxa-content-server/app/tests/spec/models/reliers/relier.js

```diff
@@ -89,6 +90,77 @@ describe('models/reliers/relier', function () {
     });
   });
 
+  describe('fetchSubscriptionProductName', () => {
+    const subscriptionProductId = 'prod_8675309';
+    const subscriptionProductName = '123 Done Pro';
+    const config = {
+      authServerUrl: 'http://example.com',
+    };
+    const productNameUrl = `${config.authServerUrl}/v1/oauth/subscriptions/productname?productId=${subscriptionProductId}`;
+
+    let respJson;
+    beforeEach(() => {
+      respJson = sinon.stub().callsFake(() =>
+        Promise.resolve({
+          /* eslint-disable camelcase */
+          product_name: subscriptionProductName,
+          /* eslint-enable camelcase */
+        })
```

Comment:
nit: could just be `sinon.stub().resolves({product_name: subscriptionProductName})`